services/business/istio_validations.go

```diff
@@ -65,6 +65,23 @@ func (in *IstioValidationsService) GetNamespaceValidations(namespace string) (mo
 	return models.NamespaceValidations{namespace: runObjectCheckers(objectCheckers)}, nil
 }
 
+func (in *IstioValidationsService) GetWorkloadValidations(namespace string, workload string) (models.IstioValidations, error) {
+	selector, err := in.k8s.GetDeploymentSelector(namespace, workload)
```

Comment:
very good!
Workload may not be a deployment, but we don't care much about that yet. So that's fine

Reply:
We should refactor this and the Workload model with the new objects in the future